README.md

```diff
@@ -16,7 +16,7 @@ fonts in your project and use the CSS classes listed below.
 
 Or just link to it using [RawGit](//rawgit.com) (served via [MaxCDN](//www.maxcdn.com)'s network):
 
-	<link href="//cdn.rawgit.com/Lukas-W/font-linux/v0.5/assets/font-linux.css" rel="stylesheet">
+	<link href="//cdn.rawgit.com/Lukas-W/font-linux/master/assets/font-linux.css" rel="stylesheet">
```

Comment:
That URL is cached, so replacing the version number with `master` will get you an outdated version. See #3.
Rawgit says:
> It's best to use a specific tag or commit hash in the URL (not a branch). Files are cached permanently after the first request.